geospaas_processing/cli/download.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script for cli of downloading repo. This script accepts the argunemt from sys.args by the
+help of python "argparse" and download the dataset files based on the input arguments.
+"""
+import argparse
+import os
+import subprocess
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+import json
+import django
+import ast
+
+from dateutil.relativedelta import relativedelta
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'geospaas_processing.settings')
+django.setup()
+from geospaas.catalog.models import Dataset
+import geospaas_processing.downloaders as downloaders
+
+
+def main(ar):
+    cumulative_limitation = {}
+    limitation_list = json.loads(ar.limitation)
+    for limitation in limitation_list:
+        cumulative_limitation.update(limitation)
+    if ar.rel_time_flag:
+        designated_begin = datetime.now().replace(tzinfo=tzutc()) + relativedelta(
+            hours=-abs(int(ar.begin)))
+        designated_end = datetime.now().replace(tzinfo=tzutc())
+    else:
+        designated_begin = datetime.strptime(ar.begin, "%Y.%m.%d:%H").replace(
```

Comment:
@akorosov 
I have used `%Y.%m.%d:%H` because the user may want to download the data from a specific hour of a day!
DO you mean only just by searching the days is enough? (by using `%Y-%m-%d`).
If we use days only, then we can not execute the code every 12 hours or every 8 hours! Because the code can distinguish between the hours! We can only run it once a day.
I think this cli is supposed to work based on hours in addition to days. Minutes and seconds are too much details and they may just become boring when working with them. What do you decide @akorosov ?

Reply:
Done in the next commit